gemfiles/rails5.1.gemfile

```diff
@@ -2,4 +2,3 @@
 eval_gemfile('./common.rb')
 
 gem 'activerecord', '~> 5.1.0'
-gem 'sqlite3', '~> 1.3', '>= 1.3.6'
```

Comment:
ActiveRecord has a version restriction hidden in its SQLite adapter. I usually copy the version restriction verbatim into each matching Gemfile, to avoid a later release of sqlite3 from suddenly breaking the tests. I vote that we keep these lines.

Reply:
E.g. https://github.com/rails/rails/blob/5-1-stable/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L10

If ActiveRecord’s database adapters were separate gems, they would each have a hard version restriction on their dependencies.